src/js/page_sections/Navmenu.js

```diff
@@ -1,115 +1,164 @@
 import React, { Component } from 'react';
-import request from 'graphql-request'
-import CloseSVG from 'js/svg/Close';
-import I18nNavLink from 'js/modules/I18nNavLink'
+import request from 'graphql-request';
+
 import allTopicPagesQuery from 'js/queries/allTopicPagesQuery';
+import I18nNavLink from 'js/modules/I18nNavLink';
+import ExternalLink from 'js/modules/ExternalLink';
+import navigation from '__tmpdata/navigation';
+
+import ChevronDownSVG from 'js/svg/ChevronDown';
+import CloseSVG from 'js/svg/Close';
+import AirplaneSVG from 'js/svg/Airplane';
+import PlusSVG from 'js/svg/Plus';
+import MinusSVG from 'js/svg/Minus';
+import citySealImg from 'images/coa_seal.png';
+
 
 class Navmenu extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      nav: [],
-      isLoaded: false
+      openSection: null,
     };
+    this.DESKTOP_BREAKPOINT = 1080;
   }
 
   focusOnClose = () => {
     this.refs.closeTrigger && this.refs.closeTrigger.focus();
   }
 
-  componentDidMount () {
-    request(
-      `${process.env.CMS_API}`,
-      allTopicPagesQuery,
-    ).then(res => {
-      this.setState({
-        data: res.allTopics,
-        isLoaded: true
-      });
-    }).catch(err => console.log(err));
-  }
-
-  componentDidUpdate() {
-    if (this.props.isOpen) {
+  componentDidUpdate(prevProps) {
+    if (this.props.isOpen && !prevProps.isOpen) {
       this.focusOnClose();
     }
   }
 
-  getOverlayClassName = () => {
-    let className = `coa-Navmenu__overlay`;
-    if (this.props.isOpen) {
-      className = `${className} ${className}--open`;
+  toggleMobileSublist = (e, openSectionId) => {
+    if (window.innerWidth < this.DESKTOP_BREAKPOINT) {
```

Comment:
Does something like this work for SSR? Do we care about that now?

Reply:
hmm, interesting point, but this function only executes on a click event, so I don't think we need to expect this to be node/server-safe.

And if it works for React-Static, then I don't see why it wouldn't work in SSR. And it currently works in the PR deploy. 

But good eye to think about using the `window` with care. I wasn't thinking about that when I wrote this.